Tests/ASPerformanceTestContext.mm

```diff
@@ -54,6 +57,10 @@ - (instancetype)init
   return self;
 }
 
+- (NSDictionary<NSString *,ASPerformanceTestResult *> *)results {
+  return _results;
+}
+
```

Comment:
Note: C++ was complaining that our ivar is NSMutableDictionary, not NSDictionary.